app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt

```diff
@@ -161,7 +161,7 @@ class BindableAdapterTest {
         assertThat(recyclerView.childCount).isEqualTo(1)
       }
       // Perform onView() verification off the the main thread to avoid deadlocking.
-      onView(atPosition(R.id.test_recycler_view, 0))
+      onView(atPosition(R.id.test_recycler_view, position = 0))
```

Comment:
add named argument for recyclerview id too.  
```suggestion
      onView(
        atPosition(
          recyclerViewId = R.id.test_recycler_view,
          position = 0
        )
      )
```

Reply:
ditto for all

Reply:
added named arguments for recyclerViewId : https://github.com/oppia/oppia-android/pull/3153/commits/431cf43838b96dca47cfe23de07f460bd53d4ed3